requirements.txt

```diff
@@ -2,6 +2,7 @@
 # Version updates managed by pyup.io
 
 attrdict==2.0.1
+betelgeuse==1.7.0
```

Comment:
I think this should go into `requirements-optional.txt` (Ideally we refactor anyway to use setuptools extra_requires groups),  as its not directly related to actually executing pytest from robottelo.

Reply:
removing from `requirements.txt`